controller/src/main/java/org/jboss/as/controller/operations/global/GlobalOperationHandlers.java

```diff
@@ -687,7 +687,7 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
                 PathAddress target = aliasEntry.convertToTargetAddress(addr.append(element));
                 PathAddress targetParent = target.subAddress(0, target.size() - 1);
                 Resource parentResource = context.readResourceFromRoot(targetParent, false);
-                if (parentResource != null && parentResource.hasChildren(target.getLastElement().getKey())) {
+                if (parentResource != null && parentResource.hasChild(target.getLastElement())) {
```

Comment:
I believe this needs logic to deal with the possibility that target.getLastElement().isWildcard() is true. If so there won't be a child at that address and it will need to be handled more like the "if" block above this "else" block.

I assume it's because of that case that the code was written the way it is, but that left the WFCORE-692 problem.